content/en/blog/grpc-csharp-future.md

```diff
@@ -1,24 +1,25 @@
 ---
 draft: true
 title: The future of gRPC in C# belongs to grpc-dotnet
-date: 2021-04-26
+date: 2021-05-01
+spelling: cSpell:ignore dotnetcore nuget Tattermusch
 author:
   name: Jan Tattermusch
   link: https://github.com/jtattermusch
 ---
 
-_TL;DR  grpc-dotnet (i.e. the
+_TL;DR  grpc-dotnet (the
```

Comment:
I checked the rendered version of the "TL;DR" paragraph and it's hard to read. Should TL;DR be followed by a line break?

Reply:
I agree that italics is difficult to read. What I've seen most often is to put TL;DR in bold (rather than use a line break).

Reply:
Sounds like a good idea.

Reply:
I've submitted that change. Let me know what you think.